src/code_80281FA0.h

```diff
@@ -7,19 +7,19 @@
 
```

Comment:
Couldn't hurt to rename this to ceremony_and_credits.h

Also, is there a need to comment out a bunch of the forward declarations?l

Reply:
I commented them out because I was changing function types and I would have to change it in .c and .h.

Will fix

Thanks for taking a look at the PR.